uuid.zig

```diff
@@ -62,15 +75,15 @@ pub const Uuid = struct {
 
     pub fn fromInt(n: u128) Uuid {
         var v: [16]u8 = undefined;
-        std.mem.writeIntBig(u128, &v, n);
+        std.mem.writeInt(u128, &v, n, .big);
         return Uuid{ .bytes = v };
     }
 
     pub fn toInt(self: Uuid) u128 {
-        return std.mem.readIntBig(u128, &self.bytes);
+        return std.mem.readInt(u128, &self.bytes, .big);
     }
 
-    fn init(comptime version: u4, bytes: [16]u8) Uuid {
+    pub fn init(comptime version: u4, bytes: [16]u8) Uuid {
```

Comment:
I think this function should probably stay private - there's not much usecase for initializing a UUID with raw bytes *and* wanting the version to be initialized (thereby overwriting some of the bytes), outside of generating a new UUID, which this library does for you.

If you want to create a UUID out of an already correctly initialized set of bytes, you can simply do `uuid.Uuid{ .bytes = bytes }`

Reply:
I mostly did this in case the user wants to generate a v4 using a custom RNG as well as for any other hackery you might want, like making up your own version or something.